### ISSUE TITLE
Import error with latest version of scipy

### ISSUE THREAD

**Issue by marcel-goldschen-ohm** [shakedzy/dython#159]
```
sys.version_info(major=3, minor=12, micro=1, releaselevel='final', serial=0)
dython.__version__ = 0.7.4
```

### Describe the bug:
Importing dython fails with latest version of scipy (1.12?).
Code to reproduce:
1. `pip install dython` alongside the latest version of `scipy` (1.12?).
2. Attempt to `import dython` raises error (I don't recall exactly, but something to do with interp module not found or similar). 

### Nonideal solution
The error is resolved if you install the prior version of `scipy==1.11.4`. So the error must be due to a change in the latest version of scipy. Would be great if dython was updated to be compatible with the latest scipy.

**Comment by shakedzy** [comment]
Duplicate of: #156 - Fix is already available by installing from source (version `0.7.5.dev`), will be released to PyPi soon.

**Comment by shakedzy** [comment]
Version 0.7.5 has been published to PyPi
